lib/shared/layers/python-sdk/python/genai_core/aurora/query.py

```diff
@@ -185,78 +187,69 @@ def query_workspace_aurora(
                 item["keyword_search_score"] = current["keyword_search_score"]
 
     unique_items = list(unique_items.values())
-    score_dict = dict({})
-    if len(unique_items) > 0:
-        passages = [record["content"] for record in unique_items]
-        passage_scores = genai_core.cross_encoder.rank_passages(
-            cross_encoder_model, query, passages
-        )
-
-        for i in range(len(unique_items)):
-            score = passage_scores[i]
-            unique_items[i]["score"] = score
-            score_dict[unique_items[i]["chunk_id"]] = score
-
-    unique_items = sorted(unique_items, key=lambda x: x["score"], reverse=True)
-
-    for record in vector_search_records:
-        record["score"] = score_dict[record["chunk_id"]]
-    for record in keyword_search_records:
-        record["score"] = score_dict[record["chunk_id"]]
+    if (config["rag"]["crossEncodingEnabled"]):
```

Comment:
It would be better that everything that is conditional to having enable crossEncodingModels is under a single `if` statement. This should be the one